webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -215,31 +265,29 @@ function createDMIntroMessage(channel: Channel, centeredIntro: string, teammate?
                 <div className='post-profile-img__container channel-intro-img'>
                     <ProfilePicture
                         src={src}
-                        size='xl'
+                        size='xl-custom-DM'
+                        status={teammate.is_bot ? '' : channel.status}
                         userId={teammate?.id}
                         username={teammate?.username}
                         hasMention={true}
                     />
                 </div>
-                <div className='channel-intro-profile d-flex'>
+                <h2 className='channel-intro__title'>
```

Comment:
There are also E2E tests that use `.channel-intro-profile` which also need to be updated

Reply:
Updated the test files